app/models/thread.php

```diff
@@ -0,0 +1,88 @@
+<?php
+class Thread extends AppModel                    
+{
+    public $validation = array('title'=>array('length'=>array('validate_between',1,30),),); 
+
+    public static function getAll($offset, $limit)                
+    {
+        $threads = array();
+        $db = DB::conn();
+        $rows = $db->rows("SELECT * FROM thread LIMIT {$offset}, {$limit}");
```

Comment:
Or maybe you can cast or validate the variable value if it is number or integer
since using placeholders in `SimpleDBI` will cast the values to string..